packages/session-replay-browser/src/config/joined-config.ts

```diff
@@ -22,41 +23,88 @@ export class SessionReplayJoinedConfigGenerator {
     });
   }
 
-  async generateJoinedConfig(sessionId?: number) {
+  async generateJoinedConfig(sessionId?: number): Promise<SessionReplayJoinedConfig> {
     const config: SessionReplayJoinedConfig = { ...this.localConfig };
     // Special case here as optOut is implemented via getter/setter
     config.optOut = this.localConfig.optOut;
+    // We always want captureEnabled to be true, unless there's an override
+    // in the remote config.
+    config.captureEnabled = true;
+    let remoteConfig: SessionReplayRemoteConfig | undefined;
     try {
+      if (!this.remoteConfigFetch) {
+        return config;
+      }
+
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
These overrides make me think this was typing as any, which maybe means that the remote config updates weren't built for you locally - after pulling v1.x, run `yarn build` in the `packages/analytics-remote-config` folder and see if this typing gets fixed then